_data/sites.json

```diff
@@ -1597,6 +1597,16 @@
         ]
     },
 
+    {
+        "name": "Curve",
+        "url": "https://www.curve.app/",
+        "difficulty": "hard",
+        "notes": "They cannot delete your account immediately as they are obliged by the Money Laundering Regulations 2017 to keep your data for 5 years after you stop using Curve.",
```

Comment:
I understand it might take some time for the data to be fully deleted, but how do you request deletion?

Reply:
I forgot to put that in. Users will have to request it via email. I am currently in the process of contacting them about this to clarify this more clearly. I am awaiting their reply.

Reply:
Ok, thanks for the info, let's keep this on hold until you get an answer then

Reply:
Any luck?